lib/oca-epak/epak/client.rb

```diff
@@ -100,22 +177,30 @@ def get_operation_codes
       # @param [String] "From date" in DD-MM-YYYY format
       # @param [String] "To date" in DD-MM-YYYY format
       # @return [Array, nil] Contains an array of hashes with NroProducto and NumeroEnvio
-      def list_shipments(cuit, from_date, to_date)
+      def list_shipments(opts = {})
```

Comment:
@cavi21 What's the goal of changing this method's signature? I don't see any new parameters within the `opts` hash.